src/results/outages.jl

```diff
@@ -67,13 +74,24 @@ function add_outage_results(m, p, d::Dict)
 	r["unserved_load_per_outage_kwh"] = round.(unserved_load_per_outage, digits=2)
 	r["storage_microgrid_upgrade_cost"] = value(m[:dvMGStorageUpgradeCost])
 	r["microgrid_upgrade_capital_cost"] = r["storage_microgrid_upgrade_cost"]
+    _n = ""
 	if !isempty(p.s.storage.types.elec) && Bool(round(value(m[:binMGStorageUsed]), digits=0))
 		r["storage_discharge_series_kw"] = value.(m[:dvMGDischargeFromStorage]).data
+        electric_storage_energy_capacity_kwh = round(value(m[Symbol("dvStorageEnergy"*_n)]["ElectricStorage"]), digits=2)
```

Comment:
More robust to use p.s.storage.types.elec instead of hardcoding "ElectricStorage" key

Reply:
Addressed in 9275b0de798fa66d1e77efbc194ca8f6879926f7.